src/index.spec.js

```diff
@@ -14,7 +14,6 @@ describe('babel plugin', () => {
   //       to starting subprocesses.
   var procPool
   beforeEach(() => { procPool = new ProcessPool })
-  afterEach(() => { procPool.destroy() })
```

Comment:
Both. I don't think it has anything to do with the change. When I put the line back, here's my mocha output:

      babel plugin
        ✓ compiles a single add event (1269ms)
    Unhandled rejection Error: channel closed
        at ChildProcess.target.send (internal/child_process.js:509:16)
        at /home/flaise/projects/sigh-babel/node_modules/process-pool/src/ProcessPool.js:17:16
        at tryCatcher (/home/flaise/projects/sigh-babel/node_modules/bluebird/js/main/util.js:26:23)
        at Promise._settlePromiseFromHandler (/home/flaise/projects/sigh-babel/node_modules/bluebird/js/main/promise.js:507:31)
        at Promise._settlePromiseAt (/home/flaise/projects/sigh-babel/node_modules/bluebird/js/main/promise.js:581:18)
        at Promise._settlePromiseAtPostResolution (/home/flaise/projects/sigh-babel/node_modules/bluebird/js/main/promise.js:245:10)
        at Async._drainQueue (/home/flaise/projects/sigh-babel/node_modules/bluebird/js/main/async.js:128:12)
        at Async._drainQueues (/home/flaise/projects/sigh-babel/node_modules/bluebird/js/main/async.js:133:10)
        at Immediate.Async.drainQueues [as _onImmediate] (/home/flaise/projects/sigh-babel/node_modules/bluebird/js/main/async.js:15:14)
        at processImmediate [as _immediateCallback] (timers.js:374:17)
        1) forwards module option to Babel module
        2) forwards optional settings to Babel module


      1 passing (6s)
      2 failing

      1) babel plugin forwards module option to Babel module:
         Error: timeout of 2500ms exceeded. Ensure the done() callback is being called in this test.
      

      2) babel plugin forwards optional settings to Babel module:
         Error: timeout of 2500ms exceeded. Ensure the done() callback is being called in this test.
      
I'm using Ubuntu 14.04 and Node 4.1.1. Perhaps recent versions of Node have breaking changes compared to your version?